pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -294,9 +318,8 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo, provisionC
 	}
 	volumes := controller.PodVolumes(provisionConfig.DataPathMap, dataDirHostPath, false)
 	failureDomainValue := osdProps.crushHostname
-	doConfigInit := true       // initialize ceph.conf in init container?
-	doBinaryCopyInit := true   // copy tini and rook binaries in an init container?
-	doActivateOSDInit := false // run an init container to activate the osd?
```

Comment:
Note for reviewers: I removed this to instead use `!osdProps.onPVC()` (i.e., OSD is on node) to determine when to use the OSD activate init container.